levelObjects/Actions.md

```diff
@@ -173,18 +176,15 @@ For the height fields the value ```0x0FFF``` indicates "don't move". Otherwise i
 
 **Random Timer Action Details** (16 byte)
 
-    0000  int32      Object index
-    0004  int32      Time limit
-    0008  int32      Activation value
-    000C  [4]byte    Unknown
-
-The ```Time limit``` specifies within which time span the given object shall be triggered randomly. The first activation
-waits until the complete time limit has elapsed, further activations are random within this limit.
-
-The ```Activation value``` must be ```0xFFFF``` or higher for this action to work.
-
-> The game has the ```Activation value``` typically set to ```0xFFFF```, ```0x10000``` or ```0x11111```. No difference has been found.
-
+	0000  int32      Object index
+	0004  int32      Delay
+	0008  int16      Count
+	000A  int16      Time span min
```

Comment:
The Count or minimum fields do not work as proposed. The max field on the other hand does add some variance to the timer (to make it actually random).